docker/k8s/pmm-client/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM vitess/base AS base
+
+FROM debian:stretch-slim
+
+# Copy CA certs for https calls
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+
+RUN apt-get update && \
```

Comment:
We should put all these RUN commands into script file, and just COPY and RUN that. Currently since these are separate layers, we're still paying for the storage of all the removed files.

Reply:
Ok, I can do that